astropy/conftest.py

```diff
@@ -19,6 +20,15 @@
 from astropy.tests.helper import enable_deprecations_as_exceptions
 
 try:
+    # This is needed to silence a warning from matplotlib caused by
+    # PyInstaller's matplotlib runtime hook.  This can be removed once the
+    # issue is fixed upstream in PyInstaller, and only impacts us when running
```

Comment:
For future reference, is there already an open issue in PyInstaller about this?

Reply:
https://github.com/pyinstaller/pyinstaller/issues/3959#issuecomment-496878862 ?

Reply:
There is also a draft PR to fix it.  It's been delayed for a bit and still seems to have some kinks to work out, but I reviewed it myself and it seems sensible.  I thought of trying to integrate it into this fix, but it seems to be a bit tricky to override existing runtime hooks, so I decided it could get too complicated https://github.com/pyinstaller/pyinstaller/pull/5006